### PR TITLE
8325,8326,8327,8328,8330,8334,8335,8336,8338,8344,8345,8347 Fix Requi…

### PR DESCRIPTION
**Pull request by GillsITWorld**
…red Race

### CODE REVIEW

File: Database/Corrections/TBC/tbcQuestFixes.lua [hunk 1]
```diff
@@ -112,15 +112,47 @@ function QuestieTBCQuestFixes:Load()
...
+            [questKeys.requiredRaces] = raceIDs.BLOOD_ELF,
+        },
+        [8334] = {
+            [questKeys.requiredRaces] = raceIDs.BLOOD_ELF,
+        },
+        [8335] = {
+        [questKeys.requiredRaces] = raceIDs.BLOOD_ELF,
+        },
+        [8336] = {
+        [questKeys.requiredRaces] = raceIDs.BLOOD_ELF,
```
Comment (BreakBB) [inline Database/Corrections/TBC/tbcQuestFixes.lua:138]:
```suggestion
            [questKeys.requiredRaces] = raceIDs.BLOOD_ELF,
```

File: Database/Corrections/TBC/tbcQuestFixes.lua [hunk 2]
```diff
@@ -112,15 +112,47 @@ function QuestieTBCQuestFixes:Load()
...
+        },
         [8330] = {
             [questKeys.preQuestSingle] = {},
+            [questKeys.requiredRaces] = raceIDs.BLOOD_ELF,
+        },
+        [8334] = {
+            [questKeys.requiredRaces] = raceIDs.BLOOD_ELF,
+        },
+        [8335] = {
+        [questKeys.requiredRaces] = raceIDs.BLOOD_ELF,
```
Comment (BreakBB) [inline Database/Corrections/TBC/tbcQuestFixes.lua:135]:
```suggestion
            [questKeys.requiredRaces] = raceIDs.BLOOD_ELF,
```

**Review by BreakBB — CHANGES_REQUESTED** [review]
Just some small code formatting issue. Good catch on these quests!

**Review by GillsITWorld — COMMENTED** [review]
Will double-check all PRs before Submitting.

**Review by BreakBB — APPROVED** [review]

**Pull request merged**